### DISCUSSION TITLE
How can i response many others user fields in jwt login ?

### DISCUSSION THREAD

**Discussion by ScrimForever** [fastapi-users/fastapi-users#1299] [Q&A]
how can I pass other user fields in the login call, without having to call another /me endpoint? 
I tried to use 

 async def on_after_login(
            self, user: User, request: Optional[Request] = None, response: Optional[Response] = None):
        return {user.email, user.is_active, user.id, user.is_verified, user.is_superuser, user.nome_empresa, user.cnpj} 
       
in user_management , but it not work

**Comment by frankie567** [comment]
The response object produced is controlled by the `AuthenticationBackend` class:

https://github.com/fastapi-users/fastapi-users/blob/ff9fae631cdae00ebc15f051e54728b3c8d11420/fastapi_users/authentication/backend.py#L42-L46

You might overload the `login` method of this class to customize the response to your need. For example (supposing you use Bearer transport):

```py
class CustomAuthenticationBackend(AuthenticationBackend):
        async def login(
            self, strategy: Strategy[User, UUID], user: user
        ) -> Response:
            token = await strategy.write_token(user)
            return JSONResponse({"access_token": token, "token_type": "bearer", "email": user.email})
```

and then use it:

```py
auth_backend = CustomAuthenticationBackend(
    name="jwt",
    transport=bearer_transport,
    get_strategy=get_jwt_strategy,
)
```
